draft-ietf-snac-simple.xml

```diff
@@ -692,12 +703,14 @@
       <t>
 	In the fourth case, the stub router MUST provide its own NAT64 service.</t>
       <t>
-	An additional complication is that there may be more than one stub router connecting the stub network to infrastructure.
-	In this case, it may be desirable to limit the number of stub routers providing NAT64 service, or it may be acceptable
-	for all stub routers to provide it. In the latter case, this should not be a problem: since each stub router is using
-	its own ULA prefix to provide NAT64, any 5-tuple that goes through a stub router's NAT64 translator will necessarily
-	have as its destination an IPv6 address in a particular NAT64 prefix, and that address will select the correct stub
-	router through which to send the packet for translation.</t>
+	An additional complication is that there may be more than one stub router connecting the stub network to infrastructure.  In
+	this case, it may be desirable to limit the number of stub routers providing NAT64 service, or it may be acceptable for all
+	stub routers to provide it. In the latter case, this should not be a problem: since each stub router is using its own ULA
```

Comment:
Maybe a paragraph break before "In the latter case"?

Reply:
Updated.